scala-sbt/README.md

```diff
@@ -0,0 +1,5 @@
+## Scala SBT tutorials
+
+This module contains articles about Scala's SBT features 
+
+- [Creating a Fat JAR using SBT](https://www.baeldung.com/scala/creating-a-fat-jar-using-sbt/)
```

Comment:
Since we don't know the final URL yet, let's either omit the file or omit this line.

Reply:
Hi @KevinGilmore , I have removed the link to the article.